packages/block-editor/src/components/url-input/index.js

```diff
@@ -101,16 +107,15 @@ class URLInput extends Component {
 	}
 
 	shouldShowInitialSuggestions() {
-		const { suggestions } = this.state;
 		const {
 			__experimentalShowInitialSuggestions = false,
 			value,
 		} = this.props;
+
 		return (
 			! this.isUpdatingSuggestions &&
 			__experimentalShowInitialSuggestions &&
-			! ( value && value.length ) &&
-			! ( suggestions && suggestions.length )
+			! ( value && value.length )
```

Comment:
I'm not totally sure what effect removing `! ( suggestions && suggestions.length )` will have on other components that consume `URLInput`. That said `shouldShowInitialSuggestions` is a recent addition so I'm hopeful the impact would/should be fairly minimal.

Reply:
I've reverted, as it was an issue related to using the reset button to update suggestions. As the reset button has been removed, this has been reverted.